python/ray/_private/worker.py

```diff
@@ -891,6 +895,8 @@ def get_objects(
             bytes: UUID of the debugger breakpoint we should drop
                 into or b"" if there is no breakpoint.
         """
+        from ray.util.insight import record_object_get
```

Comment:
Import `record_object_get` right above `for value, object_ref in zip(values, object_refs):`